repo/content/index/content_id_to_bytes.go

```diff
@@ -2,53 +2,26 @@ package index
 
 import (
 	"bytes"
-	"encoding/hex"
-
-	"github.com/kopia/kopia/repo/hashing"
 )
 
-// unpackedContentIDPrefix is a prefix for all content IDs that are stored unpacked in the index.
-const unpackedContentIDPrefix = 0xff
```

Comment:
I see how this is backwards compatible with existing indexes, and that eventually indexes would be overwritten and it would no longer matter that this was used as prefix. Nevertheless, it'd be good to have comment regarding this.

Reply:
this was never used for real indexes, so safe to remove.